src/System.ObjectModel/tests/packages.config

```diff
@@ -1,10 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="System.Collections" version="4.0.10-beta-22605" />
-  <package id="System.ObjectModel" version="4.0.10-beta-22605" />
-  <package id="System.Runtime" version="4.0.20-beta-22605" />
-  <package id="System.Threading" version="4.0.0-beta-22512" />
-  <package id="System.Threading.Tasks" version="4.0.10-beta-22512" />
+  <package id="System.Runtime" version="4.0.20-beta-22512" />
+  <package id="System.Threading" version="4.0.10-beta-22605" />
```

Comment:
I was going to ask you to make the version numbers consistent here as well but it looks like we have this inconsistency in a lot of other config files as well right now. The next time we publish all our packages we should run @nguerrera script to make the versions all consistent.